### DISCUSSION TITLE
ERROR:  You requested the 1st label, but the specified ZPL doesn't generate any labels

### DISCUSSION THREAD

**Discussion by J2012123** [porrey/Virtual-ZPL-Printer#36] [Q&A]
Hello - I am trying to print labels via the virtual ZPL from Microsoft Dynamics - my test labels print - but when I send labels to the printer from D365 I get the following error - can someone help me?

ERROR:  You requested the 1st label, but the specified ZPL doesn't generate any labels

**Comment by porrey** [comment]
Try using the online editor at http://labelary.com/viewer.html and see if it gives you any additional information.

**Comment by amsummer55** [comment]
I get this same message.  When I paste my file contents in labelary though, it shows the label with no problem.  Not sure why the vprinter isnt working
